docs/standard/exceptions/snippets/how-to-use-specific-exceptions-in-a-catch-block/csharp/catchexception.cs

```diff
@@ -29,7 +29,7 @@ public static void PromoteEmployee(Object emp)
         e.Emlevel = e.Emlevel + 1;
     }
 
-    public static void Main()
+    static void Main(string[] args)
```

Comment:
```suggestion
    static void Main()
```

No reason to add `args` if they're not being used. It's perfectly valid to have a [`void Main()`](https://docs.microsoft.com/dotnet/csharp/programming-guide/main-and-command-args/#overview).